x-pack/elastic-agent/pkg/agent/cmd/container.go

```diff
@@ -40,15 +41,19 @@ import (
 )
 
 const (
-	requestRetrySleep     = 1 * time.Second                  // sleep 1 sec between retries for HTTP requests
-	maxRequestRetries     = 30                               // maximum number of retries for HTTP requests
-	defaultStateDirectory = "/usr/share/elastic-agent/state" // directory that will hold the state data
+	requestRetrySleepEnv     = "KIBANA_REQUEST_RETRY_SLEEP"
+	maxRequestRetriesEnv     = "KIBANA_REQUEST_RETRY_COUNT"
+	defaultRequestRetrySleep = "1"                              // sleep 1 sec between retries for HTTP requests
```

Comment:
Can we make this `1s` instead and have time parsing? This makes it possible to pass in any time values if needed and makes it more explicity.